cli/script/command-executor.ts

```diff
@@ -965,8 +972,16 @@ export var releaseReact = (command: cli.IReleaseReactCommand): Promise<void> =>
             throw new Error(`Entry file "${entryFile}" does not exist.`);
         }
     }
+    
+    if (command.appStoreVersion && semver.validRange(command.appStoreVersion) === null) {
+        throw new Error("Please use a semver-compliant target binary version range, for example \"^1.0.3\".");
```

Comment:
Maybe add another semver range example? Something like '...for example: "1.0.0", "*" or "^1.2.3")'?